config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: a9bca5cbc2c7719c10133623bbe336851fb70269cf5cbf2a90b677ee80d3e68ba6f8fd0b5f379fa73f4aa1729c556a1e0cb211ac47f804ab8813c0519578ca10
+
+test:
+  secret_key_base: d07b10fa2256be6c9c2f6dcbd85061d8adb6623889e0ddf3e1651fa113c19022e659ead26f0fb6a5a767c7621085965d00c1b0d5291f37ee7668e893b50a2eac
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
We should probably maintain a list somewhere of .env variables we need to set .env.example perhaps?